docs/proposals/004-endpoint-picker-protocol/README.md

```diff
@@ -10,6 +10,9 @@ The EPP MUST implement the Envoy
 [external processing service](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ext_proc/v3/ext_proc.proto) protocol.
 
 ## Endpoint Subset
+
+[REQUEST: Data Plane -> EPP]
```

Comment:
I like the way you introduced clear understanding of when each point is used in the flow.
can we be consistent with  naming selection? 
the header says [Data Plane -> EPP] but the first sentence says [proxy communicate to EPP]

Reply:
Thanks, I've made this consistent.